fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/AggregateValueTest.java

```diff
@@ -202,6 +208,29 @@ void testTupleSumCount() {
         });
     }
 
+    @Nullable
+    public static List<Long> collectOnBits(@Nullable byte[] bitmap) {
+        if (bitmap == null) {
+            return null;
+        }
+        int start = 0;
```

Comment:
is this necessary?

Reply:
so the old planner was using this function in test, this could guarantee that we behave exactly like the older planner.